DOCS-1826: Link between generics (#2507)
viamrobotics/docs: docs/components/generic/_index.md

However, if you want to use an LED display for example, you need very different functionality that isn't currently exposed in any API.
Instead, you can use the generic component API to add support for your unique type of hardware, like LED displays, to your machine.

Use generic for a {{< glossary_tooltip term_id="modular-resource" text="modular resource" >}} model that represents a unique type of hardware.

> If your use case calls for adding new high-level software functionality, rather than supporting new hardware components, consider using the [generic service](/registry/advanced/generic/) instead.

Reviewer: ```suggestion
If you are adding new high-level software functionality, rather than supporting new hardware components, use the [generic service](/registry/advanced/generic/) instead.
```
[opt] For brevity, and then can we be more directive since they should absolutely use service? Are we worried about sounding too bossy?

Author: Agreed!
